src/components/ComplainantInfoView.js

```diff
@@ -25,9 +25,9 @@ class ComplainantInfoView extends React.Component {
       complainantNameValid: true,
       sectionValid: false,
       didClickNav: this.props.location.state
-          ? this.props.location.state.didClickNav
-          : false,
-      currentPage: location.hash.substr(2, 10)
+        ? this.props.location.state.didClickNav
+        : false,
+      currentPage: parseInt(location.hash.substr(2), 10)
```

Comment:
`parseInt(location.hash.substr(2), 10)` is repeated in every section component. We can probably figure out a way to not have to repeat this every time. 

One possible solution that comes to mind is to use a wrapper component for each "section" of the form, and that wrapper can pass in some of the common props that each component depends on.

Another thought is to look into how `react-router` could help us. The `location` prop from `react-router` is useful, and we might be able to take advantage of it for passing along the `currentPage`.

Speaking of `react-router`, I'd recommend using the `location` prop over the global `location` object simply because `react-router` is what handles all navigation related stuff, and the `location` prop is the source-of-truth, whereas the global `location` object may not always be totally correct.

Reply:
Updated to use history.location and extracted a helper function so that it's a little less repetitive.